pdfgen.h

```diff
@@ -158,6 +158,12 @@ struct pdf_path_operation {
  */
 #define PDF_TRANSPARENT (uint32_t)(0xffu << 24)
 
+/*!
+ * Convert big endian value to little endian value.
+ */
+#define BIG_TO_LITTLE(x) (((x) >> 24) | (((x) & 0x00FF0000) >> 8) | (((x) & 0x0000FF00) << 8) | ((x) << 24))
```

Comment:
We probably shouldn't assume the host is little endian, so we might be better off using endian.h (and moving this inside of pdfgen.c). However since endian.h is non-standard (ie: not available on Windows/Mac), perhaps we'll have to roll our own fragment of it at the top of pdfgen.c. Similar to this: https://github.com/mikepb/endian.h/blob/master/endian.h